cnf-tests/hack/build-test-bin.sh

```diff
@@ -16,27 +16,23 @@ export GOFLAGS="${GOFLAGS:-"-mod=vendor"}"
 export PATH=$PATH:$GOPATH/bin
 DONT_REBUILD_TEST_BINS="${DONT_REBUILD_TEST_BINS:-false}"
 
-if ! which ginkgo; then
-	echo "Installing ginkgo tool from vendor"
-	go install -mod=vendor github.com/onsi/ginkgo/v2/ginkgo
-fi
-
-mkdir -p bin
-
-function build_and_move_suite {
-  suite=$1
-  target=$2
-
-  if [ "$DONT_REBUILD_TEST_BINS" == "false" ] || [ ! -f "$target" ]; then
-    ginkgo build ./testsuites/"$suite"
-    mv ./testsuites/"$suite"/"$suite".test "$target"
-  fi
-}
-
-build_and_move_suite "e2esuite" "./bin/cnftests"
-build_and_move_suite "configsuite" "./bin/configsuite"
-build_and_move_suite "validationsuite" "./bin/validationsuite"
-
 if [ "$DONT_REBUILD_TEST_BINS" == "false" ] || [ -f ./cnf-tests/bin/mirror ]; then
+  echo "Building mirror ... "
   go build -o ./bin/mirror mirror/mirror.go
+  retVal=$?
+  if [ $retVal -ne 0 ]; then
```

Comment:
those checks aren't necessary because you have `set -e` at the beginning of the file.

Reply:
you're right.
thanks for the catch.